duckduckgo/Android
# Updated app icon silhouette.

Task/Issue URL: https://app.asana.com/0/414730916066338/1201337702286803/f

### Description

Updated the silhouette app icon to match the latest design and to looks the same in settings and after it is selected.

### Steps to test this PR

- install / upgrade from this branch
- go to Settings and tap on App Icon
- check the silhouette app icon (the last one on the screen) looks as expected -> has a bow tie and you can see the eyes
- select the silhouette app icon and check that the app icon is now the same as the one you selected
- there are 2 types of icons, round and square. For the square one use an old OS version, up to Android N.

### UI changes

| Square icon Before  | Square icon After |
| ------ | ----- |
![Screenshot_1636455323](https://user-images.githubusercontent.com/7963079/140911808-2056f744-2ba3-43db-9a05-05ea7751e770.png)|![Screenshot_1636455192](https://user-images.githubusercontent.com/7963079/140911867-8cb1da0e-486b-4fe0-a445-1fbd65bdcaaf.png)|

| Round icon Before  | Round icon After |
| ------ | ----- |
![round_icon_before](https://user-images.githubusercontent.com/7963079/140912863-17419143-1fae-4b9c-a775-bb3b6a8a9b2d.png)|![round_icon_after](https://user-images.githubusercontent.com/7963079/140912902-7f3058c9-c4e4-41be-8d86-2bc5c45b5a54.png)|

| Settings Before  | Settings After |
| ------ | ----- |
![settings_before](https://user-images.githubusercontent.com/7963079/140912993-862ba6a9-1c40-43e2-a504-d925e01e7f0b.png)|![settings_after](https://user-images.githubusercontent.com/7963079/140913007-cb6f3373-074d-4e93-83fe-fa11fc34cc18.png)|
